defaults.yaml

```diff
@@ -0,0 +1,28 @@
+qabel:
+    index:
+        # SECURITY WARNING: don't run with debug turned on in production!
+        DEBUG: False
+        ALLOWED_HOSTS:
+          - '*'
+
+        # SECURITY WARNING: keep the secret key used in production secret!
+        SECRET_KEY: '=tmcici-p92_^_jih9ud11#+wb7*i21firlrtcqh$p+d7o*49@'
+
+        # This key has no big impact on anything, it is just a key the clients use for transmitting requests
```

Comment:
Even if the server private key is only used to verify the authenticity of the requesting user, it should be protected since the DH is symmetric: if an attacker gets in possession of the server private key it can impersonate all users.
I would suggest to add a comment on keeping the private key confidential or to remove the `has no big impact on anything` part.